contracts/rust/src/lib.rs

```diff
@@ -64,6 +64,7 @@ impl NonFungibleTokenBasic {
     }
 }
 
+#[near_bindgen]
```

Comment:
what fails if we don't have it?

Reply:
you can't call the functions, this "exposes" it publicly basically
we'll get `MethodNotFound` or something like that